XMonad/Actions/WindowBringer.hs

```diff
@@ -90,6 +93,37 @@ gotoMenu' cmd = gotoMenuConfig def { menuArgs = [], menuCommand = cmd }
 gotoMenuArgs' :: String -> [String] -> X ()
 gotoMenuArgs' cmd args = gotoMenuConfig def { menuCommand = cmd, menuArgs = args }
 
+-- | Pops open a dmenu with window titles. Choose one, and it will be copied into your current workspace.
+copyMenu :: X ()
```

Comment:
I can't help but wonder if this indicates we need some refactoring so we don't have to define five versions of everything. (No, I'm not suggesting you do it!)